libraries/AP_Logger/LogStructure.h

```diff
@@ -1792,7 +1820,9 @@ LOG_STRUCTURE_FROM_AHRS \
     { LOG_WINCH_MSG, sizeof(log_Winch), \
       "WINC", "QBBBBBfffHfb", "TimeUS,Heal,ThEnd,Mov,Clut,Mode,DLen,Len,DRate,Tens,Vcc,Temp", "s-----mmn?vO", "F-----000000" }, \
     { LOG_PSC_MSG, sizeof(log_PSC), \
-      "PSC", "Qffffffffffff", "TimeUS,TPX,TPY,PX,PY,TVX,TVY,VX,VY,TAX,TAY,AX,AY", "smmmmnnnnoooo", "F000000000000" }
+      "PSC", "Qffffffffffff", "TimeUS,TPX,TPY,PX,PY,TVX,TVY,VX,VY,TAX,TAY,AX,AY", "smmmmnnnnoooo", "F000000000000" }, \
+    { LOG_PSCZ_MSG, sizeof(log_PSCZ), \
+      "PSCZ", "Qfffffffff", "TimeUS,TPZ,PZ,DVZ,TVZ,VZ,DAZ,TAZ,AZ,ThO", "smmnnnooo-", "F000000000" }
```

Comment:
I think the multiplier should be "None" if we don't have a unit.  I think notionally it could also be percentage *0.01 IIRC.

```suggestion
      "PSCZ", "Qfffffffff", "TimeUS,TPZ,PZ,DVZ,TVZ,VZ,DAZ,TAZ,AZ,ThO", "smmnnnooo-", "F00000000-" }
```

Should probably add a sanity check for that.

Reply:
I guess it really is a percentage.  I stole the entry from the identical CTUN.ThO field.. but maybe I'll change both to be percentages

Reply:
@peterbarker I've changed it to a percentage.  Can you confirm that you're ok with it now?

Reply:
Still LGTM.